entrypoint.sh

```diff
@@ -40,13 +40,13 @@ else
 		make \
 			BUILD_LOG="$BUILD_LOG" \
 			IGNORE_ERRORS="$IGNORE_ERRORS" \
-			"package/$PKG/download" V=s || \
+			"package/$PKG/download" V="$V" || \
```

Comment:
Sorry now I remember why this was hardcoded to `s`: You should always have the full "debug" log for the checks, but you likely don't want the the full debug log if you build a ton of packages.

Please remove this commit again.

Reply:
okay, done